databases/week2/sql_Week2Part4.sql

```diff
@@ -0,0 +1,28 @@
+CREATE DATABASE company;
+
+USE company;
+
+CREATE TABLE employees(
+id INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+first_name VARCHAR(45) NOT NULL,
+last_name VARCHAR(45)NOT NULL,
+age INT(10),
+email VARCHAR(45) NOT NULL UNIQUE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE department(
```

Comment:
Does one department only have one employee?

Reply:
Hmm. I just added a foreign key as department_id.

Reply:
Do you think I need something else

Reply:
If the deparment table as an `employee_id` foreign key, that means it can only have one employee in each department. You cannot store more than one value in the `employee_id` column. 

Maybe try doing what i've been doing with excel on the classes: create a few employees, a few departments, and then try to add two employees to the same department ;)

Reply:
Ohh yes... I don't know how I missed that.

Reply:
And the excel thing definitely makes it easier to see it

Reply:
:)
If one employee can only belong to one department this is correct. But, you need to have a many-to-many relationship in the database (it was in the exercise description), and you don't have one as it is.

Also, if you create the tables in this order you'll get an error, you cannot create the foreign keys before creating the tables they reference.